### ISSUE TITLE
video will not play from control panel

### ISSUE THREAD

**Issue by Vrak** [magdesign/PocketVJ#6]
I am not able to start videos from the control panel. I have tried placing a test video file in `/media/internal` and `/media/internal/video`.

I am able to play videos using omxplayer-sync over SSH, and I can reset the RPi 1 from the network control panel. I've installed all dependencies and followed the suggestions found in: https://github.com/magdesign/PocketVJ/issues/2

Any idea why I might be unable to start videos via the control panel?

**Comment by magdesign** [comment]
Video must be stored in /media/internal/*

If you run this command via ssh

`/var/www/sync/startmaster.py`

is the video starting?
did you make `sudo chmod a+x -R /var/www/sync/`

**Comment by Vrak** [comment]
The video did not start. `/var/www/sync/startmaster.py `outputs a number like 1080, 1058, 1016 each time I run it.

**Comment by magdesign** [comment]
and if you try to start it with:
/usr/bin/omxplayer-sync -mu -o both /media/internal/*



is there any reason why you dont use the preconfigured image which is available as free download?

**Comment by Vrak** [comment]
Yes, that image is for a 16gb disk. Currently I only have an 8gb SD available for this. 

`/usr/bin/omxplayer-sync -mu -o both /media/internal/*` gives no output over ssh and no change on the projector or screen.

**Comment by magdesign** [comment]
are you sure you installed the omxplayer-sync script in the right location and made it executable?
you should be able to start a video with: /usr/bin/omxplayer-sync /media/internal/*

**Comment by Vrak** [comment]
omxplayer-sync was in` /usr/bin/` but i noticed that omxplayer was still there too, so I removed that and re-installed omxplayer-sync, (carefully following Turing machines instructions), rebooted, then ran `/usr/bin/omxplayer-sync -mu -o both /media/internal/*` again.

and again, the output to my projector changed from the terminal after boot to a blank blackscreen. It SHOULD be playing the synctest.mp4 file from Turingmachine.
